common/src/auth/auth.service.ts

```diff
@@ -88,7 +92,13 @@ export class AuthService<User extends AbstractUser> {
     this.resetRemember();
     this.userService.resetProfile();
 
-    this.router.navigate([this.authConfig.unauthenticatedRoute ?? AuthService.DEFAULT_UNAUTHENTICATED_ROUTE]);
+    const redirectRoute = (this.authConfig.unauthenticatedRoute === undefined)

```

Comment:
sorry, I'm not sure that it differs from the current implementation:

```ts
this.router.navigate([this.authConfig.unauthenticatedRoute ?? AuthService.DEFAULT_UNAUTHENTICATED_ROUTE]);
```

They don't seem to be any different, but the current implementation is much shorter.

https://www.typescriptlang.org/play?#code/MYewdgzgLgBAJgUwGYEMCuAbKA1FG0IwC8MA5IqplKQNwBQoksAbngcTGmBQJZgJx6DcBBAYEAOgwgA5gApW+QgH5l8ZOiy4lAShpA

if we want to be able to disable the redirect, then I would suggest another implementation:

1. add a property `disableRedirectAfterUnauthorize?: false` in the config
2. check it here